assets/config.php

```diff
@@ -83,6 +83,7 @@
         Stancl\Tenancy\Bootstrappers\FilesystemTenancyBootstrapper::class,
         Stancl\Tenancy\Bootstrappers\QueueTenancyBootstrapper::class,
         Stancl\Tenancy\Bootstrappers\BatchTenancyBootstrapper::class,
+        Stancl\Tenancy\Bootstrappers\MailTenancyBootstrapper::class, // Queueing mail requires using QueueTenancyBootstrapper with $forceRefresh set to true
```

Comment:
I found out that while testing, TenancyServiceProvider was getting registered _before_ adding the bootstrapper to the `tenancy.bootstrappers` config. Because of that, the bootstrapper wasn't stateful (it didn't get bound as a singleton), and the config wasn't getting reverted properly.

I tried if uncommenting the bootstrapper in the config will fix that issue, and it did. I also couldn't think of a good reason to comment out `MailTenancyBootstrapper`.

@stancl, do you think there's a good reason to keep the bootstrapper commented out? Also, do you know of a solution for adding and registering the bootstrappers in tests properly? I thought about manually making the bootstrapper a singleton, but I wasn't sure if making the bootstrapper a singleton is everything that's happening to the bootstrappers in the provider's `register` method.

Reply:
I think we bind the Redis bootstrapper as a singleton somewhere. Could that be a similar case to this?

Reply:
I found where the Redis bootstrapper gets bound as a singleton (https://github.com/archtechx/tenancy/blob/master/tests/TestCase.php#L115 – the line also has a todo that says `use proper approach eg config for singleton registration`), and yeah, that seems like a very similar case

Reply:
https://github.com/archtechx/tenancy/pull/989/commits/62619e6738d45e0dbd2f1de627ed86f40a0f05f8